tensorflow/python/estimator/estimator.py

```diff
@@ -260,7 +266,10 @@ def get_variable_value(self, name):
       ValueError: If the Estimator has not produced a checkpoint yet.
     """
     _check_checkpoint_available(self.model_dir)
-    return training.load_variable(self.model_dir, name)
+    if isinstance(name, six.string_types):
+        return training.load_variable(self.model_dir, name)
+    else:
+        return {k:traning.load_variable(self.model_dir, _check_string_or_not(v)) for v in name}
```

Comment:
There's a typo here, this will not work. 

Please fix the code and add a test to make sure this actually works as intended.

Reply:
Good catch, I will fix it.